xbmc/utils/FileExtensionProvider.cpp

```diff
@@ -207,6 +235,10 @@ void CFileExtensionProvider::SetAddonExtensions(AddonType type)
           fileFolderExtensions.push_back(ext.first);
       }
     }
+    // Invalidate dependent cached extensions lists
+    m_musicExtensions = {};
+    m_pictureExtensions = {};
+    m_fileFolderExtensions = {};
```

Comment:
`m_subtitlesExtensions` is missing here?!

Reply:
What about `m_discStubExtensions`?

Reply:
same answer as other similar code - those 2 don't depend on addons.